<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Star Voyager: Defender</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">

   <property name="styleSheet">
    <string notr="true"/>
   </property>

   <!-- PLAY BUTTON -->
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>

    <!-- Qt5-safe font -->
    <property name="font">
     <font>
      <family>Orbitron</family>
      <bold>true</bold>
      <italic>false</italic>
     </font>
    </property>

    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: "Orbitron";
    font-size: 13px;
    color: white;

    background-color: transparent;
    border: 3px solid #b56dff;
    border-radius: 20px;

    padding: 10px 20px;
}

QPushButton:hover {
    border-color: #33E3FF;
    background-color: rgba(0, 200, 255, 0.15);
}

QPushButton:pressed {
    border-color: #00AACC;
    background-color: rgba(0, 200, 255, 0.35);
}</string>
    </property>

    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>

   <!-- EXIT BUTTON -->
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>

    <property name="font">
     <font>
      <family>Orbitron</family>
      <bold>true</bold>
      <italic>false</italic>
     </font>
    </property>

    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: "Orbitron";
    font-size: 13px;
    color: white;

    background-color: transparent;
    border: 3px solid #ffcc4d;
    border-radius: 20px;

    padding: 10px 20px;
}

QPushButton:hover {
    border-color: #33E3FF;
    background-color: rgba(0, 200, 255, 0.15);
}

QPushButton:pressed {
    border-color: #00AACC;
    background-color: rgba(0, 200, 255, 0.35);
}</string>
    </property>

    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>

   <!-- SETTINGS BUTTON -->
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>

    <property name="font">
     <font>
      <family>Orbitron</family>
      <bold>true</bold>
      <italic>false</italic>
     </font>
    </property>

    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: "Orbitron";
    font-size: 13px;
    color: white;

    background-color: transparent;
    border: 3px solid #ff914d;
    border-radius: 20px;

    padding: 10px 20px;
}

QPushButton:hover {
    border-color: #33E3FF;
    background-color: rgba(0, 200, 255, 0.15);
}

QPushButton:pressed {
    border-color: #00AACC;
    background-color: rgba(0, 200, 255, 0.35);
}</string>
    </property>

    <property name="text">
     <string>SETTINGS</string>
    </property>
   </widget>

  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>24</height>
    </rect>
   </property>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
